spec/models/transformation_mapping_item_spec.rb

```diff
@@ -100,4 +106,82 @@
       end
     end
   end
+
+  # ---------------------------------------------------------------------------
+  # Network Validation
+  # ---------------------------------------------------------------------------
+  context "Network validation" do
+    let(:ems_vmware) { FactoryBot.create(:ems_vmware) }
+    let(:vmware_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => ems_vmware) }
+
+    let(:ems_redhat) { FactoryBot.create(:ems_redhat) }
+    let(:redhat_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => ems_redhat) }
+
+    let(:ems_ops) { FactoryBot.create(:ems_openstack) }
+    let(:cloud_tenant) { FactoryBot.create(:cloud_tenant_openstack, :ext_management_system => ems_ops) }
+
+    # source network
+    context "source vmware network" do
+      let(:src_vmware_host) { FactoryBot.create(:host_vmware, :ems_cluster => vmware_cluster) }
+      let(:src_switch) { FactoryBot.create(:switch, :host => src_vmware_host) }
```

Comment:
I don't think you can actually set `:host` on a switch this way since there is a host_switches join table not a host_id on the record.  You can do `FactoryBot.create(:switch, :hosts => [src_vmware_host])` though.